Tests/ConnectLibraryTests/ConnectConformance/AsyncAwaitConformance.swift

```diff
@@ -361,14 +368,7 @@ final class AsyncAwaitConformance: XCTestCase {
             let expectation = self.expectation(description: "Stream completes")
             let stream = client.failStreamingOutputCall()
             try stream.send(Grpc_Testing_StreamingOutputCallRequest.with { proto in
```

Comment:
This can just be `try stream.send(Grpc_Testing_StreamingOutputCallRequest())` since an empty array is the default value